R/pkg/R/mllib_regression.R

```diff
@@ -100,6 +120,12 @@ setMethod("spark.glm", signature(data = "SparkDataFrame", formula = "formula"),
               print(family)
               stop("'family' not recognized")
             }
+            # Handle when family = statmod::tweedie()
+            if (tolower(family$family) == "tweedie" && !is.null(family$variance)) {
```

Comment:
This part only handles the case when `statmod::tweedie` is specified: it retrieves the `var.power` and `link.power` and construct a list with family name and link name to be used. 
The check for non-null `variance` is to skip handling the "fake" family. All we need when specifying `family = "tweedie"` is just a list with family name and link name.